ext/datasource/helper.go

```diff
@@ -85,23 +68,15 @@ func NewFlowRulesHandler(converter PropertyConverter) PropertyHandler {
 	return NewDefaultPropertyHandler(converter, FlowRulesUpdater)
 }
 
-// SystemRulesJsonConverter provide JSON  as the default serialization for list of system.SystemRule
-func SystemRulesJsonConverter(src []byte) (interface{}, error) {
+// SystemRulesJsonArrayParser provide JSON  as the default serialization for list of system.SystemRule
+func SystemRulesJsonArrayParser(src []byte) (interface{}, error) {
```

Comment:
keep naming convention consistent.
Maybe "SystemRuleJsonArrayParser" be better?